### ISSUE TITLE
Some text in the UI has a wrong color in when using a high contrast theme on Windows

### ISSUE THREAD

**Issue by calixteman** [mozilla/pdf.js#14728]
STR:
 - switch to `Night Sky` in Windows accessibility setting.
 - open a pdf
 - some text in the top bar are white when it should be yellow
 ![image](https://user-images.githubusercontent.com/5641725/160659502-5f069e1a-292c-462a-9b8d-773bb39f87df.png)
 - open the zoom menu and the menu items are a bit hard to distinguish
 ![image](https://user-images.githubusercontent.com/5641725/160659732-e4a83e3b-ee35-4657-af32-27d4871a1f73.png)

**Comment by calixteman** [comment]
It has been fixed thanks to #15438.
